test/unit_test/systems/azureml/output_metrics/pipeline_output/named-outputs/accuracy/metric_result.json

```diff
@@ -1 +1,7 @@
-{"result": 0.99618}
+{
+    "accuracy-accuracy_score": {"value": 0.99618, "priority": 1, "higher_is_better": true},
+    "accuracy-f1_score": {"value": 0.99618, "priority": 1, "higher_is_better": true},
```

Comment:
Question: will all subtype metrics have `priority 1`?

Reply:
Fixed. Only one metric should have `1` as top priority.